### PR TITLE
The item overhaul!

### PR DESCRIPTION
**Pull request by MrAsbestos**
### Purpose
This PR updates and adds ""some"" sprites, adds and fixes sounds, and also fixes some issues, I would have added even more stuff, but the feature freeze is coming soon, so I decided to make this PR, I really hope I didn't broke or replaced anything, I did a small check and I think it's fine, but if you notice something off, please tell me, and I'll get to work.

### Changelog:
Oh boy this is a biiiig one..

CL: Added a PDA belt overlay.
CL: Added a stun baton belt overlay.
CL: Added a new cool stun baton sprite, and removed the other sprite.
CL: Attacking with bike horns now display an attack message that says "You HONKED (target's name) in the (limb)", just like TG!
CL: Fixed the emergency oxygen tank belt overlay not showing up.
CL: Added a crowbar belt overlay.
CL: Added a health analyzer belt overlay.
CL: Added a atmos analyzer belt overlay.
CL: The disabler now has it's proper sprite now.
CL: Fixed the disposal outlet not making sounds (still has to be set up by someone *wink wink*).
CL: All shotguns now have new awesome handheld sprites.
CL: Increased the shake distances for bombs for a more shocking explosion (20).
CL: Fixed normal EVA helmets showing up as syndicate space helmets.
CL: Added syndicate minibomb handheld overlay.
CL: New beautiful jukebox sprites.
CL: Added handheld sprites for all existing medkits.
CL: All boxes now have handheld sprites.
CL: Warden's jacket now has the correct handheld overlay.
CL: All bomb suits and hoods have the correct handheld overlay.
CL: The captain's carapace now has a handheld overlay.
CL: All donuts now have a handheld overlay.
CL: The grenade now has a handheld overlay.
CL: The reflector jacket and vest now have a handheld overlay.
CL: Added missing spray sounds (1 and 3).
CL: Fixed sprayers not making spraying sounds.
CL: New telebaton sprites
CL: Added belt and handheld overlays for all types of bags.
CL: Added a normal wrapping paper handheld sprite.
CL: New awesome SM sprite.
CL: Adjustments to the spray can spraying sound (changed to the accurate version of TG).
CL: Added belt overlays for flashlights and flares.
CL: The SWAT mask now has it's correct handheld overlay.
CL:  Adjusted the syndicate magboots sprites.
CL: Adjusted the insulated gloves handheld sprite.
CL: Fixed the fuel, water, and foam tank rotations (when you went up the sprite turned right, when you went left the sprite turned down, you'll see it fixed now).
CL: Added descriptions and names for the nuclear devices and modified their beeping sounds to sound more recognizable and better (basically, it sounds like an actual bomb counting down).
CL: Fixed some handheld sprites for alcohol drinks.
CL: Fixed cola cans being able to be broken when thrown (lol).
CL: Added belt overlays for pens.
CL: Added some handheld overlays to some candy, and chip items.
CL: Adjusted the handheld overlays for the geiger counter and export scanner.
CL: Added handheld overlays for assemblies and stock parts.
CL: Welding tools now have a belt overlay.

Aaaand that's all! I would have done more, after this feature block is out, I will continue to update and add sprites :)

### CODE REVIEW

**Comment by MrAsbestos** [comment]
wut, ok, I didn't even touch any docker thing, neither the HTML thing

File: UnityProject/AddressablePackingProjects/SoundAndMusic/.vsconfig [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿{
+  "version": "1.0",
+  "components": [ 
+    "Microsoft.VisualStudio.Workload.ManagedGame"
+  ]
+}
```
Comment (PerfectTangent) [inline UnityProject/AddressablePackingProjects/SoundAndMusic/.vsconfig:6]: should this have been committed?

**Comment by ThatDan123** [comment]
Missing reference found in prefab SuperMatter component Sprite field Sprite
Missing reference found in prefab Jukebox component Sprite field Sprite
Missing reference found in prefab SuperMatterHugBox component Sprite field Sprite

**Comment by ThatDan123** [comment]
I made a PR to your branch to fix the issues:

https://github.com/MrAsbestos/unitystation/pull/2

**Comment by ThatDan123** [comment]
~Theres a issue of the sprites not working correctly in the project files, ive asked bod to take a look as i cant any obvious reason the error happens~

**Comment by ThatDan123** [comment]
Managed to find the issue and have also put it in that PR for you to merge.

**Comment by ThatDan123** [comment]
Can you go through all the sprites you've changed and make sure you havnt got empty values like this one:

When you merge my PR an error log will be generated when youre looking in a folder with a broken SO

image

**Comment by MrAsbestos** [comment]
Alright I'll check on it

File: UnityProject/AddressablePackingProjects/SoundAndMusic/.vsconfig (hunk 1 above)
Comment (MrAsbestos) [reply UnityProject/AddressablePackingProjects/SoundAndMusic/.vsconfig:6]: Hmmm, I really don't know.... but I don't think it's causing issues?

**Comment by MrAsbestos** [comment]
Do not merge by the way, I am still going to fix some stuff, but not right now, I am kind of busy.

File: UnityProject/AddressablePackingProjects/SoundAndMusic/.vsconfig (hunk 1 above)
Comment (corp-0) [reply UnityProject/AddressablePackingProjects/SoundAndMusic/.vsconfig:6]: it shouldn't but it is not causing issues either

**Comment by corp-0** [comment]
Too many changes! For the future try to make smaller PRs.

**Comment by MrAsbestos** [comment]
Oops! well then, I'll just wait then after the feature block is gone, I'll continue to rebase it from time to time, and maybe add some stuff, just a bit of stuff.

also this happens when I remove the empty elements, the sprite rotations don't really work, they stay the same

https://user-images.githubusercontent.com/65787528/112221892-3ea98200-8be5-11eb-9825-843c64256059.mp4

**Comment by MrAsbestos** [comment]
and btw most of the changes are just sprite changes, just telling ya

**Comment by MaxIsJoe** [comment]
> and btw most of the changes are just sprite changes, just telling ya

Does this fix any layering issues items used to have?

**Comment by MrAsbestos** [comment]
Er no, the only items that have rn layering issues are gloves and backpacks, and that's really it, just 2 clothing items

by sprite changes I meant adding sprites, clothing data, literally changing the sprite, that sort of stuff

**Comment by MrAsbestos** [comment]
Ready to merge, merge it when you think it's time to.

**Comment by Aranclanos** [comment]
okay there's a couple problems here, it includes a lot of non intended changes from other commits (for example all warops stuff) and its also an enourmous PR making it a pain to review making me think that it'll just get blindmerged
the nature of unreadeability unity prefabs doesn't help

maybe separate it into chunks and each one having its own pr, it's gonna help a lot with merge conflicts popping up too

**Comment by MrAsbestos** [comment]
hmm yeah....
It's gonna be a pain in the ass to remake everything again, but oh well, I did fuck up lol
or wait I think I can do something to recover some stuff..
eitherway I am closing this PR, I didn't knew that squashing existed and made this mess

**Comment by MrAsbestos** [comment]
yeah I can use "git reflog" and that stuff, nice

**Comment by MrAsbestos** [comment]
ooh wait nvm, everything is like, such a mess lol, eh little, here we go again

**Pull request closed without merging**